### PR TITLE
Open Market Tacsuits

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

I've spoken with this subject with quite a few people so far, that there's some inherent issues with the game's setting as regards to corporations, and people readily identifying certain companies as being "Syndicate Companies". It makes absolutely no sense for things like that to be completely public knowledge that X-Company is explicitly antagonistic, when publicly they would just be companies that sell products to an open market-- an open market that NanoTrasen buys from. And while we've been doing this lately with adding non-NT company names to the names and descriptions of hardsuits, one thing has been bugging me and other people for awhile. 

"If the Cybersun tacsuits are meant to be something that the Nukies literally bought on the open market and painted red. Why can't the station also purchase these suits from the same market? If a manufacturer of these suits notably declined to sell its products to another company, it would be an extremely obvious tell that they were overtly hostile to that corporation. It makes more sense if they openly sell said products at an inflated cost, so that they can put up public appearances of neutrality".

This also brings to mind that the Reverse Engineering Machine, in it's current rendition, is extremely problematic for establishing the game as a serious setting, given that it currently functions as what is essentially "Press button to commit industrial espionage". Having talked with its original creator, Rane also agreed that- especially in the context of the "Blukie Suits", this presents an issue for the game's setting presentation. It's a plot hole that makes very little sense.

So, what this PR does, is make it so that station Logistics can purchase crates containing CSA branded Tacsuits, painted in the blue, at a massively overinflated cost. These crates come with Security locks, meaning that they would be bought for use solely by the station's security. Additionally, this removes the ability for the Reverse Engineering machine to violate the intellectual property of other corporations, which in-universe would be something highly illegal under a truly neutral space law.

I actually also believe that other hardsuits and tacsuits, including ones purchased from Five-Points-Armory, Hephaeustus, etc, should all be purchasable from cargo. Technically, any weapon, hardsuit, or armor, should also be purchasable, but making it so that cargo can order a Sec-Crate with a C-20r in it would be extremely controversial, and is best left to a separate PR.

# Changelog

:cl:
- add: CSA-51a "Shanlin" and CSA-80UA "Guan-Yu" Tacsuits can now be purchased from Logistics for use by station security personnel, at an extremely overinflated price. These suits come prepainted in blue. 
- remove: Reverse Engineering machines can no longer violate the intellectual property rights of publicly traded corporations. This means that Cybersun Tacsuits; regardless of where they were obtained from or what color they were painted, can no longer be reverse engineered.

### CODE REVIEW

File: Resources/Locale/en-US/prototypes/entities/clothing/outerClothing/hardsuits.ftl [hunk 1]
```diff
@@ -74,6 +74,9 @@ ent-ClothingOuterHardsuitLuxury = HpI-20c - "Minos" hardsuit
 ent-ClothingOuterHardsuitSyndie = CSA-51a - "Shanlin" tacsuit
     .desc = A tactical combat hardsuit produced by the Cybersun-Armaments Corporation, the suit's tags indicate it provides moderate protection against most forms of damage.
     This one has been painted blood red. It feels incredibly light.
+ent-ClothingOuterHardsuitSyndieReverseEngineered = CSA-51a - "Shanlin" tacsuit
+    .desc = A tactical combat hardsuit produced by the Cybersun-Armaments Corporation, the suit's tags indicate it provides moderate protection against most forms of damage.
+    This one has been painted blue. It feels incredibly light.
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/prototypes/entities/clothing/outerClothing/hardsuits.ftl:79]:
```suggestion
        This one has been painted blue. It feels incredibly light.
```

File: Resources/Locale/en-US/prototypes/entities/clothing/outerClothing/hardsuits.ftl [hunk 2]
```diff
@@ -87,6 +90,9 @@ ent-ClothingOuterHardsuitSyndieCommander = CSA-54c - "Tianming" tacsuit
 ent-ClothingOuterHardsuitJuggernaut = CSA-80UA - "Guan Yu" tacsuit
     .desc = The pride and joy of the Cybersun-Armaments Corporation, named after an ancient Sol' War God. Commonly known throughout the galaxy as a "Juggernaut".
     Matching its bulky appearance, it protects against all forms of damage. It feels VERY heavy.
+end-ClothingOuterHardsuitJuggernautReverseEngineered = CSA-80UA - "Guan Yu" tacsuit
+    .desc = The pride and joy of the Cybersun-Armaments Corporation, named after an ancient Sol' War God. Commonly known throughout the galaxy as a "Juggernaut".
+    Matching its bulky appearance, it protects against all forms of damage. It feels VERY heavy.
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/prototypes/entities/clothing/outerClothing/hardsuits.ftl:95]:
```suggestion
        Matching its bulky appearance, it protects against all forms of damage. It feels VERY heavy.
```

File: Resources/Prototypes/Catalog/Cargo/cargo_security.yml [hunk 3]
```diff
@@ -77,3 +77,23 @@
...
+
+- type: cargoProduct
+  id: SecurityGuanYuTacsuit
+  icon:
+    sprite: Nyanotrasen/Clothing/OuterClothing/ReverseEngineering/juggernaut.rsi
+    state: icon
+  product: CrateSecurityGuanYuTacsuit
+  cost: 30000
+  category: cargoproduct-category-name-security
+  group: market
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Cargo/cargo_security.yml:99]:
```suggestion
  group: market

```

File: Resources/Prototypes/Catalog/Cargo/cargo_security.yml [hunk 4]
```diff
@@ -77,3 +77,23 @@
...
   group: market
+
+- type: cargoProduct
+  id: SecurityShanlinTacsuit
+  icon:
+    sprite: Nyanotrasen/Clothing/OuterClothing/ReverseEngineering/syndicate.rsi
+    state: icon
+  product: CrateSecurityShanlinTacsuit
+  cost: 17500
+  category: cargoproduct-category-name-security
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Cargo/cargo_security.yml:88]: hate

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Review by FoxxoTrystan — APPROVED** [review]
Man... antags need to be buffed or just more antag.

**Pull request merged**